src/libsystemd/sd-netlink/netlink-types.c

```diff
@@ -16,6 +16,7 @@
 #include <linux/if_link.h>
 #include <linux/if_tunnel.h>
 #include <linux/l2tp.h>
+#include <linux/if_macsec.h>
```

Comment:
Keeping the include list alphabetically sorted would need this to move upwards.

Reply:
yes thanks